### PR TITLE
fix: save for later button update in the sidebars

### PR DESCRIPTION
**Pull request by WojtekTheWebDev**
Save for later logic update

## Description

Save for later button has been removed from the cart sidebar.
The same button has been changed to add to cart on the wishlist sidebar.
Wishlist icon in app header are showing if there are some items in the wishlist.

## Related Issue

VBC-373

## How Has This Been Tested?

Locally

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Review by dhargitai — APPROVED** [review]
Nice one

File: packages/theme/components/CartSidebar.vue [hunk 1]
```diff
@@ -60,6 +60,9 @@
                     />
                   </div>
                 </template>
+                <template #actions>
+                  <div></div>
```
Comment (scernisevs) [inline packages/theme/components/CartSidebar.vue:64]: just a question: why would we need an empty div?

Comment (WojtekTheWebDev) [reply packages/theme/components/CartSidebar.vue:64]: @scernisevs There is no way of removing or hiding the "save for later" button, I have to place empty div there

**Review by scernisevs — APPROVED** [review]
LGTM :) thank you!

**Pull request merged**